Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -622,6 +679,29 @@ extension SPNoteListViewController {
             notesListController.object(at: indexPath) as? Note
         }
     }
+
+    open override func setEditing(_ editing: Bool, animated: Bool) {
+        super.setEditing(editing, animated: animated)
+
+        // Reloading data ensures that no cells are selected and the current data is up to date
+        let rows = tableView.indexPathsForVisibleRows
+        tableView.reloadRows(at: rows!, with: .automatic)
+
+        // Dispatches to asyncAfter because reloadData will override the animations if not complete
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1, execute: {
```

Comment:
I had poked around for some other ways to get around this, but I didn't find any while also reloading the rows.  This was in there because if we batch deleted some rows and then returned to edit mode, the cells that were deleted are the only ones that would enter edit mode.  

The fix was to reload the cells before changing edit mode, and that overrides the animations.  But I will take a look and see what I can find